workers/unity/Assets/Gdk/Core/GameObjectRepresentation/GameObjectDispatcherCallbacksSystem.cs

```diff
@@ -0,0 +1,70 @@
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    /// <summary>
+    ///     Gathers incoming dispatcher ops and invokes callbacks on relevant GameObjects.
+    /// </summary>
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    public class GameObjectDispatcherCallbacksSystem : ComponentSystem
+    {
+        private MutableView view;
```

Comment:
If we move the function that finds the generated classes we can remove this reference.

Reply:
Moved the GameObjectTranslationsDictionary into the system.